pages/common/rmlint.md

```diff
@@ -7,9 +7,9 @@
 
 `rmlint {{path/to/directory1 path/to/directory2 ...}}`
 
-- Check for space wasters, preferably keeping files in tagged directories (after the double slash):
+- Check for duplicates bigger than 1MB, preferably keeping files in tagged directories (after the double slash):
 
-`rmlint {{path/to/directory}} // {{path/to/original_directory}}`
+`rmlint -s {{1MB}} {{path/to/directory}} // {{path/to/original_directory}}`
```

Comment:
I'd prefer to keep the placeholder here and change the description to something like "Check for duplicates bigger than the specified size". This would make it clearer which parts of the command line are fixed strings and which ones are not.